### ISSUE TITLE
"examples"field is not an Array of Maps

### ISSUE THREAD

**Issue by meunierfrederic** [tehmantra/saunter#11]
[In reference documentation](https://www.asyncapi.com/docs/specifications/2.0.0/#a-name-messageobject-a-message-object), one can see that Message.examples field is a [Map[string,any]].

However, in Message Class, Examples member is a IDictionary<string,object>.
Generated documentation validation fails (online editor [here](https://playground.asyncapi.io/)).

With the following in Message.cs line 42
```
        [JsonProperty("examples")]
        public IList<IDictionary<string, object>> Examples { get; set; } = new List<IDictionary<string, object>>();

```
generated documents are valid.

**Comment by tehmantra** [comment]
Fixed by #12

**Comment by meunierfrederic** [comment]
Thanks Michael.
Will you publish this fix in Saunter NuGet Package ?

**Comment by tehmantra** [comment]
Yep. Published in NuGet 0.1.0-alpha-42
https://www.nuget.org/packages/Saunter/0.1.0-alpha-42

**Comment by meunierfrederic** [comment]
Test was successful with NuGet 0.1.0-alpha-42 and 0.1.0-alpha-44.
Thanks Michael
